adapters/conversant/conversant.go

```diff
@@ -1,291 +1,177 @@
 package conversant
 
 import (
-	"bytes"
-	"context"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
-	"net/http"
-
 	"github.com/mxmCherry/openrtb"
 	"github.com/prebid/prebid-server/adapters"
 	"github.com/prebid/prebid-server/errortypes"
-	"github.com/prebid/prebid-server/pbs"
-	"golang.org/x/net/context/ctxhttp"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
 )
 
 type ConversantAdapter struct {
 	http *adapters.HTTPAdapter
```

Comment:
The `http *adapters.HTTPAdapter` field is part of the old interface and should be removed. Doesn't look like you're referencing it anyway.